Source/Samples/SamplesManager.cpp

```diff
@@ -178,8 +178,15 @@ void SamplesManager::Setup()
 #if MOBILE
     engineParameters_[EP_ORIENTATIONS] = "Portrait";
 #endif
-    if (!engineParameters_.contains(EP_RESOURCE_PREFIX_PATHS))
+    if (!engineParameters_.contains(EP_RESOURCE_PREFIX_PATHS)) 
+    {
         engineParameters_[EP_RESOURCE_PREFIX_PATHS] = ";..;../..";
+        if (GetPlatform() == PlatformId::MacOS ||
+            GetPlatform() == PlatformId::iOS)
+            engineParameters_[EP_RESOURCE_PREFIX_PATHS] = ";../Resources;../..";
```

Comment:
I wonder if we really need special case here tbh. Sure its not needed on other platforms.. but it does not hurt either. I would just go with this single line instead of this entire block. @eugeneko what do you think?